TCCTF_Structures_v3/Entities/Structures/Altar/Dragonfriend/AltarDragonfriend.as

```diff
@@ -493,10 +423,8 @@ void onCommand(CBlob@ this, u8 cmd, CBitStream@ params)
 							}
 							break;
 						}
+						updateStonks(this, quantity);
```

Comment:
updateStonks(this, (-2*action+1)*quantity); // -1 or 1 * quantity

Reply:
is gud